integration_test/tests/power_bi_tests.rs

```diff
@@ -88,6 +89,7 @@ mod integration {
 
             // Generate the connection string and add a null terminator because PowerBi uses SQL_NTS for the length
             in_connection_string = generate_default_connection_str();
+            in_connection_string.push_str("DATABASE=integration_test;");
```

Comment:
That logic in the default connection str function makes sense. The reason I added this was to account for the initial [evergreen failure](https://spruce.mongodb.com/task/mongosql_odbc_driver_ubuntu2004_asan_integration_test_patch_d78bd74fe454f48a55ad32daa2675ac2d097cfd1_63cb1c6fd1fe07422f8ffd83_23_01_20_22_57_51/logs?execution=0) when I opened as a draft, though it is possible I misdiagnosed. Is there somewhere I can check where the `ADF_TEST_LOCAL_DB` variable is set? I don't see it in the `evergreen.yaml`

Reply:
ahhhh, of course, because the change is in the draft PR [here](https://github.com/mongodb/mongo-odbc-driver/pull/59/files#diff-18df3e2ba4f534c8a4b8c1d123fb89bdeaa0229641993ac244e63ba3a106fc7c) ....
You need to add this line in "generate expansions" : `export ADF_TEST_LOCAL_DB="${adf_test_local_db}"`

Reply:
ah that makes sense -- will add this